modules/base/rendering/renderableplane.cpp

```diff
@@ -78,6 +79,31 @@ namespace {
         "aspect ratio of the content. Otherwise it will remain in the given size."
     };
 
+    constexpr openspace::properties::Property::PropertyInfo ScaleByDistanceInfo = {
+        "ScaleByDistance",
+        "Scale By Distance",
+        "Decides whether the plane should automatically adjust in size to match the "
+        "distance to the camera. Otherwise it will remain in the given size."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo ScaleRatioInfo = {
+        "ScaleRatio",
+        "Scale Ratio",
+        "The scale ratio for scaling a plane by distance to camera."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo ScaleByDistanceMaxHeightInfo = {
```

Comment:
```suggestion
    constexpr openspace::properties::Property::PropertyInfo ScaleByDistanceMaxHeightInfo =
    {
```
Exceeding the line width